packages/camera/camera/example/lib/main.dart

```diff
@@ -62,53 +63,53 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
   @override
   void initState() {
     super.initState();
-    WidgetsBinding.instance.addObserver(this);
+    WidgetsBinding.instance?.addObserver(this);
+
     _flashModeControlRowAnimationController = AnimationController(
       duration: const Duration(milliseconds: 300),
       vsync: this,
     );
     _flashModeControlRowAnimation = CurvedAnimation(
-      parent: _flashModeControlRowAnimationController,
+      parent: _flashModeControlRowAnimationController!,
       curve: Curves.easeInCubic,
     );
     _exposureModeControlRowAnimationController = AnimationController(
       duration: const Duration(milliseconds: 300),
       vsync: this,
     );
     _exposureModeControlRowAnimation = CurvedAnimation(
-      parent: _exposureModeControlRowAnimationController,
+      parent: _exposureModeControlRowAnimationController!,
       curve: Curves.easeInCubic,
     );
     _focusModeControlRowAnimationController = AnimationController(
       duration: const Duration(milliseconds: 300),
       vsync: this,
     );
     _focusModeControlRowAnimation = CurvedAnimation(
-      parent: _focusModeControlRowAnimationController,
+      parent: _focusModeControlRowAnimationController!,
       curve: Curves.easeInCubic,
     );
   }
 
   @override
   void dispose() {
-    WidgetsBinding.instance.removeObserver(this);
-    _flashModeControlRowAnimationController.dispose();
-    _exposureModeControlRowAnimationController.dispose();
+    WidgetsBinding.instance?.removeObserver(this);
+    _flashModeControlRowAnimationController?.dispose();
+    _exposureModeControlRowAnimationController?.dispose();
     super.dispose();
   }
 
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
     // App state changed before we got the chance to initialize.
```

Comment:
Let's set a local variable for `controller` as the first line and use it everywhere after this, so we don't need the `!` and `?` everywhere.